### PR TITLE
Adding nuclick and classification bundles to zoo

### PR DESCRIPTION
**Pull request by SachidanandAlle**
Signed-off-by: Sachidanand Alle <sachidanand.alle@gmail.com>

Fixes # .
Adding nuclick and classification bundles to zoo

### Description
- Nuclick bundle
- Classification of Nuclei bundle

Both of them are trained over ConSeP dataset
All the metrics and details are recorded as part of the README

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [x] In-line docstrings updated.
- [x] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [x] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [x] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [x] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [x] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [x] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Comment by SachidanandAlle** [comment]
verified with MonaiLabel for both infer and train (single and multiple gpu)

File: models/pathology_nuclei_classification/configs/inference.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
+{
+    "imports": [
+        "$import glob",
+        "$import json",
+        "$import pathlib",
+        "$import os"
+    ],
+    "bundle_root": "/workspace/data/pathology_nuclei_classification",
```
Comment (yiheng-wang-nv) [inline models/pathology_nuclei_classification/configs/inference.json:8]: Hi @SachidanandAlle , here could the bundle root be changed as[ this ticket](https://github.com/Project-MONAI/model-zoo/issues/247) suggest? We planned to standardize the way to define `bundle_root`, thanks!

Comment (yiheng-wang-nv) [reply models/pathology_nuclei_classification/configs/inference.json:8]: same for `train.json` and the other bundle.

File: models/pathology_nuclei_classification/configs/metadata.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    "description": "A pre-trained model for Nuclei Classification within Haematoxylin & Eosin stained histology images",
+    "authors": "MONAI team",
+    "copyright": "Copyright (c) MONAI Consortium",
+    "data_source": "consep_dataset.zip from https://warwick.ac.uk/fac/cross_fac/tia/data/hovernet",
+    "data_type": "png",
+    "image_classes": "RGB channel data, intensity scaled to [0, 1]",
+    "label_classes": "single channel data",
+    "pred_classes": "4 channels OneHot data, channel 0 is Other, channel 1 is Inflammatory, channel 2 is Epithelial, channel 3 is Spindle-Shaped",
+    "eval_metrics": {
+        "f1_score": 0.8
```
Comment (yiheng-wang-nv) [inline models/pathology_nuclei_classification/configs/metadata.json]: I saw F1 is 0.85 on validation set, thus should the number be the same here?

File: models/pathology_nuclick_annotation/configs/metadata.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    "description": "A pre-trained model for Nuclei Classification within Haematoxylin & Eosin stained histology images",
+    "authors": "MONAI team",
+    "copyright": "Copyright (c) MONAI Consortium",
+    "data_source": "consep_dataset.zip from https://warwick.ac.uk/fac/cross_fac/tia/data/hovernet",
+    "data_type": "png",
+    "image_classes": "RGB channel data, intensity scaled to [0, 1]",
+    "label_classes": "single channel data",
+    "pred_classes": "2 channels OneHot data, channel 1 is nuclei, channel 0 is background",
+    "eval_metrics": {
+        "mean_dice": 0.96
```
Comment (yiheng-wang-nv) [inline models/pathology_nuclick_annotation/configs/metadata.json]: Similar question, I did not find the same number in readme, thus does the number wrong here?

File: models/pathology_nuclick_annotation/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+- -ve signal channel (other nuclei)
+
+### Output: 2 channels
+ - 0 = Background
+ - 1 = Nuclei
+
+![](images/train_in_out.jpeg)
+
+## Scores
+This model achieves the following F1 score on the validation data provided as part of the dataset:
```
Comment (yiheng-wang-nv) [inline models/pathology_nuclick_annotation/docs/README.md]: Here should it be Dice rather than F1?

File: models/pathology_nuclei_classification/configs/metadata.json (hunk 2 above)
Comment (SachidanandAlle) [reply models/pathology_nuclei_classification/configs/metadata.json]: you are right.. to many places to add these.. missed to fix them in metadata.json

File: models/pathology_nuclei_classification/configs/inference.json (hunk 1 above)
Comment (SachidanandAlle) [reply models/pathology_nuclei_classification/configs/inference.json:8]: on `bundle_root` i will update accordingly over another PR once we arrive a conclusion regarding the same. hope that's not a blocker to merge the PR

File: models/pathology_nuclei_classification/configs/metadata.json (hunk 2 above)
Comment (SachidanandAlle) [reply models/pathology_nuclei_classification/configs/metadata.json]: Fixed

File: models/pathology_nuclick_annotation/docs/README.md (hunk 4 above)
Comment (SachidanandAlle) [reply models/pathology_nuclick_annotation/docs/README.md]: Fixed

File: models/pathology_nuclick_annotation/configs/metadata.json (hunk 3 above)
Comment (SachidanandAlle) [reply models/pathology_nuclick_annotation/configs/metadata.json]: Fixed

**Review by yiheng-wang-nv — APPROVED** [review]
Hi @wyli @Nic-Ma , this PR looks good to me. May also need your help to review it, thanks in advance!

**Review by wyli — APPROVED** [review]
